cmd/initapp/initapp.go

```diff
@@ -204,7 +197,7 @@ func RunInitApp(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	PrintInitializationSummary(newApp.Name, newApp.AlternateId, newApp.ID, orgID, *mc.ProjectAlternateId)
+	PrintInitializationSummary(newApp.Name, newApp.AlternateId, newApp.ID, orgID, projectID)
```

Comment:
A note here that you are passing project ID instead of alternate ID, which I am guessing is intentional but calling out just in case. It seems to work either way as this just gets used to spit out the URL, and the URL seems handle either.